core/base-service/base-graphql.spec.js

```diff
@@ -12,15 +12,8 @@ const dummySchema = Joi.object({
 }).required()
 
 class DummyGraphqlService extends BaseGraphqlService {
-  static get category() {
-    return 'cat'
-  }
-
-  static get route() {
-    return {
-      base: 'foo',
-    }
-  }
+  static category = 'cat'
+  static route = { base: 'foo' }
```

Comment:
Looks like this will let us express things a bit more succinctly :+1: Will we still be able to use eslint to enforce a consistent ordering of these like we did for the functions?

Reply:
Looks like yes!

```
/Users/pnm/code/shields/core/base-service/base-graphql.spec.js
  16:3  error  Expected static property category to come before static property route  sort-class-members/sort-class-members
```